src/SmartComponents/taskDetailsHelpers.js

```diff
@@ -0,0 +1,47 @@
+import { dispatchNotification } from '../Utilities/Dispatcher';
```

Comment:
I'm naming it this because it will become a helper file for everything involving task details. I could change it to completedTaskDetailsHelpers, but it felt like a long name.